[2021-08-07 14:25:00,783] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: standardized_pipeline.create_standardized_table_partsupp 2021-08-07T14:20:00+00:00 [queued]>
[2021-08-07 14:25:00,813] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: standardized_pipeline.create_standardized_table_partsupp 2021-08-07T14:20:00+00:00 [queued]>
[2021-08-07 14:25:00,814] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-07 14:25:00,815] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-07 14:25:00,816] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-07 14:25:00,844] {taskinstance.py:1107} INFO - Executing <Task(MySqlOperator): create_standardized_table_partsupp> on 2021-08-07T14:20:00+00:00
[2021-08-07 14:25:00,860] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'standardized_pipeline', 'create_standardized_table_partsupp', '2021-08-07T14:20:00+00:00', '--job-id', '93', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/standardized_pipeline.py', '--cfg-path', '/tmp/tmpv81c9zuw', '--error-file', '/tmp/tmpdnef4171']
[2021-08-07 14:25:00,862] {standard_task_runner.py:77} INFO - Job 93: Subtask create_standardized_table_partsupp
[2021-08-07 14:25:00,853] {standard_task_runner.py:52} INFO - Started process 5052 to run task
[2021-08-07 14:25:00,986] {logging_mixin.py:104} INFO - Running <TaskInstance: standardized_pipeline.create_standardized_table_partsupp 2021-08-07T14:20:00+00:00 [running]> on host ab70d6dafbc6
[2021-08-07 14:25:01,110] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=standardized_pipeline
AIRFLOW_CTX_TASK_ID=create_standardized_table_partsupp
AIRFLOW_CTX_EXECUTION_DATE=2021-08-07T14:20:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-07T14:20:00+00:00
[2021-08-07 14:25:01,113] {mysql.py:70} INFO - Executing: CREATE TABLE IF NOT EXISTS standardized.PARTSUPP (
  `PART_KEY` int NOT NULL,
  `SUPPLIER_KEY` int NOT NULL,
  `PARTSUPP_AVAILQTY` int NOT NULL,
  `PARTSUPP_SUPPLYCOST` int NOT NULL,
  `PARTSUPP_COMMENT` text NOT NULL,
  `INGESTION_TIMESTAMP` datetime DEFAULT CURRENT_TIMESTAMP
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
[2021-08-07 14:25:01,143] {base.py:78} INFO - Using connection to: id: dwh_id. Host: dwh, Port: 3306, Schema: , Login: root, Password: ***, extra: {}
[2021-08-07 14:25:01,163] {dbapi.py:204} INFO - Running statement: CREATE TABLE IF NOT EXISTS standardized.PARTSUPP (
  `PART_KEY` int NOT NULL,
  `SUPPLIER_KEY` int NOT NULL,
  `PARTSUPP_AVAILQTY` int NOT NULL,
  `PARTSUPP_SUPPLYCOST` int NOT NULL,
  `PARTSUPP_COMMENT` text NOT NULL,
  `INGESTION_TIMESTAMP` datetime DEFAULT CURRENT_TIMESTAMP
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;, parameters: None
[2021-08-07 14:25:01,169] {dbapi.py:212} INFO - Rows affected: 0
[2021-08-07 14:25:01,194] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=standardized_pipeline, task_id=create_standardized_table_partsupp, execution_date=20210807T142000, start_date=20210807T142500, end_date=20210807T142501
[2021-08-07 14:25:01,261] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-07 14:25:01,306] {local_task_job.py:149} INFO - Task exited with return code 0
